odkx-src/sync-endpoint-user-instructions.rst

```diff
@@ -7,25 +7,67 @@
   subdomain
 
 
+Users and Groups
+=================
+
+A user needs to be assigned to an ODK-X group to gain permissions to access the system. More information about groups and the roles is available in :ref:`Data Permission Filters<data-permission-filters>` section.
```

Comment:
ODK-X group and access the system are a bit vague here. Which system(s)? Is there more specific language that can be used? Maybe something more like "A user needs to be assigned to a group within ODK-X sync-endpoint to set their permissions and roles for ODK-X apps."

Reply:
Yes Okay. I will modify the line.